src/core/server.ts

```diff
@@ -60,6 +61,7 @@ export class Server<C extends BaseContext> {
   mockDBConnection: boolean = false;
   resolversPath: string[];
   schema?: GraphQLSchema;
+  introspectionEnabled: boolean = true;
```

Comment:
Can we just call this `introspection`? The `Enabled` is superfluous since it's typed.

Reply:
sure 1 sec